jenkins/e2e-image/e2e-runner.sh

```diff
@@ -94,6 +95,11 @@ if [[ "${JENKINS_SOAK_MODE:-}" == "y" ]]; then
   #e2e_go_args+=(--save="gs://${JENKINS_SOAK_PREFIX}")
   # TODO FIX ME DO NOT SUBMIT AAAAAAAAAAAAAAAAAAA
   e2e_go_args+=(--save="gs://fejternetes/soak")
+elif [[ "${FEDERATION_UP:-}" != "true" && -n "${FEDERATION_CLUSTERS:-}" ]]; then
```

Comment:
@fejta yeah, it should be after that stanza. I had left this PR in a half-baked state due to the `kubetest --save` changes. I will fix it now.